features/guidance/turn-lanes.feature

```diff
@@ -5,7 +5,7 @@ Feature: Turn Lane Guidance
         Given the profile "car"
         Given a grid size of 20 meters
 
-    @bug
+    @simple
```

Comment:
simple is only the classification, 1:1 matching of turn lanes :), not simple to handle